### DISCUSSION TITLE
Table and graph component need to be added.

### DISCUSSION THREAD

**Discussion by massifcoder** [resend/react-email#1036] [Ideas]
### Goals

1. Can send short reports to user.
2. Can get idea of big data through small chart.
3. More information through less data.

### Background

Users want more data in less space. Tables and charts play important role in this.

### Proposal

Should add components where can send tables and various graphs using mails.

**Comment by chevroletdivisa** [comment]
Any update on this? If I want to fetch API data and send within a table, there isn't a <Table> component, right? @gabrielmfern
